### DISCUSSION TITLE
How did you create the .exe file

### DISCUSSION THREAD

**Discussion by gigidesy** [TreborNamor/TradingView-Machine-Learning-GUI#38] [Q&A] [answered]
Hi Roman,
First of all many thanks to make this available for everyone. 
I've one question, since I've tried to modify your project for my scope but I could not find the way to create a new .exe file.
I've tried with pyinstaller, but when I execute the .exe file it closes right after the a command prompt window is started.

It would be great if you could teach me on how you have created the .exe in the zip.

Many thanks and regards,

**Comment by TreborNamor** [comment, marked as answer]
hello, i created the executable file with pyinstaller. I used this code "pyinstaller --onefile TradeViewGUI.py" in the terminal. This youtube video should help create your executable. 

https://www.youtube.com/watch?v=UZX5kH72Yx4

**Reply by gigidesy** [reply]
Many thanks for your reply, appreciate it.
